10-angular/custom_directives/readme.md

```diff
@@ -493,18 +502,18 @@ To start, make a second directive in your Javascript just by copying and pasting
 
 ```js
 (function(){
-  var directives = angular.module('grumbleDirectives',[]);
-  directives.directive('grumbleShow', function(){
+  angular
+  .module("grumbles")
+  .directive('grumbleShow', function(){
```

Comment:
minor, but maybe the last pass of this LP should be cleaning up single vs double quotes. mixing back and forth between them.